src/sdk/pynni/nni/trial.py

```diff
@@ -43,15 +43,34 @@
 
 
 def get_next_parameter():
-    """Returns a set of (hyper-)paremeters generated by Tuner.
-    Returns None if no more (hyper-)parameters can be generated by Tuner."""
+    """
+    Get the hyper paremeters generated by tuner. For a multiphase experiment, it returns a new group of hyper
+    parameters at each call of get_next_parameter. For a non-multiphase (multiPhase is not configured or set to False),
+    it returns hyper parameters only on the first call for each trial job, it returns None since second call.
+
+    Returns
+    -------
+    dict
+        A dict object contains the hyper parameters generated by tuner, the keys of the dict are defined in
+        search space.
+        Returns None if no more hyper parameters can be generated by tuner.
```

Comment:
When will a trial start if tuner has no more parameters?

Reply:
Updated some wording.   A trial starts before get_next_parameter is called inside trial. For non-multiphase trial, get_next_parameter returns None since the second call,  what is the suggestion?